backend/core/src/auth/services/user-csv-exporter.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable, Scope } from "@nestjs/common"
+import dayjs from "dayjs"
+import { Pagination } from "nestjs-typeorm-paginate"
+import { CsvBuilder } from "../../applications/services/csv-builder.service"
+import { User } from "../entities/user.entity"
+
+@Injectable({ scope: Scope.REQUEST })
+export class UserCsvExporterService {
+  constructor(private readonly csvBuilder: CsvBuilder) {}
+
+  exportFromObject(users: Pagination<User>): string {
+    const userObj = users.items.reduce((obj, user) => {
+      const status = []
```

Comment:
I found it to be a little confusing here to call this status, when it's the role and there is also a status column

Reply:
oops, that was a mistake. I'll update it